tests/__snapshots__/help.test.js.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`--help attribute shows help 1`] = `
-"\`index.js [options]\` will run Jest unit tests in SFDX workspace
+"\`lwc-jest [options]\` will run Jest unit tests in SFDX workspace
```

Comment:
Please have Jody review the text. Ie verb tense.

Reply:
will run -> runs
Where possible, use present tense instead of future tense. Present tense is easier to understand, even for native speakers.